src/components/AssetCondition/AssetConditionActionComponent.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ActionComponent from '../../components/ActionComponent';
+
+const AssetConditionActionComponent = () => (
+  <ActionComponent />
```

Comment:
Why not use this component directly instead of creating a component that doesn't even pass props to it? In my opinion, `AssetConditionActionComponent` is unnecessary

Reply:
Hey @sachungo, I was looking at how the other components have implemented the `ActionComponent`, for instance, in the [AssetTypesAction](https://github.com/AndelaOSP/art-dashboard/blob/develop/src/components/AssetTypesAction.jsx) component. It seems necessary to have it later when implementing functionality for viewing or editing a single condition.

Reply:
Noted. Thanks for clarifying